app/models/planning_application.rb

```diff
@@ -225,6 +226,18 @@ def secure_change_url(application_id, secure_token)
     end
   end
 
+  def invalid_documents_without_change_request
+    documents.invalidated.reject { |x| document_change_requests.where(old_document: x).any? }
```

Comment:
we probably want to exclude archived documents from this, otherwise we might send a change request for an archived document? We could even use the `invalid_documents` method you have below.